tests/integration_tests/test_visualisations.py

```diff
@@ -319,10 +384,81 @@ def test_visualisation_compare_classifiers_from_prob_output_saved(csv_filename):
         except OSError as e:  # if failed, report it back to the user
             print("Error: %s - %s." % (e.filename, e.strerror))
 
+def test_visualisation_compare_classifiers_from_pred_npy_output_saved(
+        csv_filename
+):
+    """Ensure pdf and png figures from the experiments can be saved.
+
+    Predictions are loaded from npy file.
+    :param csv_filename: csv fixture from tests.fixtures.filenames.csv_filename
+    :return: None
+    """
+    input_features = [
+        text_feature(vocab_size=10, min_len=1, representation='sparse'),
+        categorical_feature(
+            vocab_size=10,
+            loss='sampled_softmax_cross_entropy'
+        )
+    ]
+    output_features = [categorical_feature(vocab_size=2, reduce_input='sum')]
+
+    # Generate test data
+    rel_path = generate_data(input_features, output_features, csv_filename)
+    encoder = 'cnnrnn'
```

Comment:
This is not used anywhere besides the next line so no need to declare a variable. Just do this instead
`input_features[0]['encoder'] = 'cnnrnn'`